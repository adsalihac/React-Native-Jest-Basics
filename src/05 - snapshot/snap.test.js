import React from 'react'
import { findWithType , findAllWithType } from 'react-shallow-testutils'
import ShallowRenderer from 'react-test-renderer/shallow';
import Snap from './snap'


const renderSnap = () => {
    renderer = new ShallowRenderer();
    renderer.render(
     <Snap />
    );
    return renderer.getRenderOutput();
  };

describe('05 - snapshot', () => {

  beforeEach(() => {
    jest.clearAllMocks();
  });

    test('renders correctly', () => {
         const result = renderSnap();
        expect(result).toMatchSnapshot();
    })
    
    


});
