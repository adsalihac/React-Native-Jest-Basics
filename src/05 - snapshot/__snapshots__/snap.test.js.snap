// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`05 - snapshot renders correctly 1`] = `
<View>
  <Text>
    ddd
  </Text>
</View>
`;
